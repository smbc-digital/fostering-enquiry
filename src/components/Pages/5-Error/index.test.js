import { React } from '../../../helpers/SetupTest'
import { FailurePage } from './index'
import renderer from 'react-test-renderer'

describe('FailurePage', () => {
    describe('snapshot', () => {
		it('should render correctly',() => { 
			// Arrange
			const history = { block: jest.fn() }
			const context = { formHeader: 'Contact the fostering team'}

			// Act
			const tree = renderer
			.create(<FailurePage history={history} context={context} />)
			.toJSON()
		
			// Assert
			expect(tree).toMatchSnapshot()
		})
	})
})
