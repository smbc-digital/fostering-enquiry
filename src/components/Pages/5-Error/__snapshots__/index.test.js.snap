// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FailurePage snapshot should render correctly 1`] = `
<div
  className="system-failure"
>
  <h1>
    Contact the fostering team
  </h1>
  <div
    className="icon"
  >
    <span
      aria-hidden="true"
      className="fa fa-5x fa-exclamation"
    />
  </div>
  <h2>
    We're having some problems with this form
  </h2>
  <p>
    We're working on it and the form will be back as soon as possible
  </p>
  <p>
    <a
      className="button-primary"
      href="/"
    >
      Start again
    </a>
  </p>
</div>
`;
